static/styles/subscriptions.css

```diff
@@ -137,8 +139,8 @@ h4.user_group_setting_subsection_title {
     .member_list_container,
     .subscriber_list_container {
         position: relative;
-        /* 2*45px (settings header) + 38px(tab-container row) + 20px (margin for .inner-box) + 134px (add user input and search widget area) = 282px */
-        max-height: calc(95vh - 282px);
+        /* 2*45px (settings header) + 38px(tab-container row) + 20px (margin for .inner-box) + 134px (add user input and search widget area) + 10px extra = 292px */
+        max-height: calc(min($stream_settings_max_height, 95vh) - 292px);
```

Comment:
We just missed there was a `max-height` for the overlay too.

Reply:
Why do we have a *comment* explaining the calculation, when we have `calc()` and we can *do* the calculation?

Why is “10px extra” needed?

Reply:
10px extra is to accommodate for really long names that can overflow into the next line. Anyway, I am trying a flex approach to see how feasible it is and hopefully not a lot of rework.